conduct.md

```diff
@@ -116,32 +116,42 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+When a conflict between students arises, it is imperative to deal with the situation in a respectful, yet effective way.
+Communication is a great tool to prevent or defuse conflicts between students, team members, or teams as a whole. According
+to Conway's Law, the final product mirrors the methods of communication that went into the product's development. So if
+conflicts occur and prevent communication, it will likely lower the quality of the final product. For these reasons, it is
+crucial to adhere to the codes of conduct relating to conflicts that are outlined below.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should first try to resolve conflicts amongst themselves, practicing humility, respect, and trust.
+- Students should talk out the conflict amongst their teammates.
+- Students should address the conflict as soon as it arises.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not needlessly escalate the conflict.
+- Students should not refuse to resolve the conflict
+- Students should not purposely cause conflicts.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the class conduct guide as specified by the unacceptable conduct sections. It is important that the conduct guide is upheld and acceptable conduct is followed in order to support a positive learning environment in the class and online.
 
-#### Acceptable Conduct
+When a violation is committed, the student is given a penalty for the total amount of violations committed during the semester.  The only exception to this is that Honor Code violations, and the penalties associated with them, are handled and investigated by the Honor Committee.
 
-- TODO
-- TODO
-- TODO
+If you believe that a violation was reported incorrectly or unfairly, it is possible to meet with the professor teaching the class to discuss a possible appeal for the violation.  After discussion, the professor will make the final decision about the violation.
 
-#### Unacceptable Conduct
+Depending on the total amount of violations, the penalty given is decided based on a leveled system as follows:
 
-- TODO
-- TODO
-- TODO
+#### Level 1 Infraction
+
+Students who commit one violation will be recorded by the professor and warned orally.
+
+#### Level 2 Infraction
+
+Students who commit violations 3 times or more in total will be required to have a meeting with the professor to talk about their current infractions and develop a plan to avoid future ones.
```

Comment:
I think that in "3 times or more in total", the "in total" is a bit redundant and unnecessary.